src/Access/ContactFormsAccessCheck.php

```diff
@@ -9,36 +9,58 @@
 use Drupal\Core\Cache\CacheableMetadata;
 use Drupal\Core\Routing\Access\AccessInterface;
 use Drupal\Core\Access\AccessResultInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Session\AccountInterface;
 
 /**
  * Route access handler for contact form routes.
  */
 class ContactFormsAccessCheck implements AccessInterface {
```

Comment:
Let's extend EntityAccess becayse that is the one that was used by default, and if we dont forbid, we call the parent access() method.